backend/server.js

```diff
@@ -0,0 +1,54 @@
+// Import necessary packages
+const express = require("express");
+const mongoose = require("mongoose");
+const cors = require("cors");
+const GameData=require("./modals/modal")
+
+// Create an Express app
+const app = express();
+
+app.use(cors());
+app.use(express.json());
+
+mongoose.set("strictQuery", false);
+
+// Set up a database connection using Mongoose
+mongoose
+  .connect(
+    "mongodb+srv://chandangowda:Gowda$123@score-keeper.nuzfevx.mongodb.net/?retryWrites=true&w=majority"
```

Comment:
Should not keep sensitive details public, should use environment variables.

Reply:
Changes made